addons/mail/static/src/js/many2many_tags_email.js

```diff
@@ -21,7 +21,7 @@ BasicModel.include({
      * @param {Object} record - an element from the localData
      * @param {string} fieldName
      * @return {Deferred<Object>} the deferred is resolved with the
-     *                            invalidPartnerIds and partnerNames
+     *                            invalidPartnerIds
      */
     _fetchSpecialMany2ManyTagsEmail: function (record, fieldName) {
```

Comment:
This function doesn't "fetch" anymore.
Either rename to `_setSpecialMany2ManyTagsEmail`, or revert to `_setInvalidMany2ManyTagsEmail`.

Reply:
@alexkuhn changes done.